File: week2/day4/week2-day4-workshop/week2-day4-workshop/Person.cs
using System;
using System.Collections.Generic;
using System.Runtime.CompilerServices;
using System.Text;

namespace week2_day4_workshop
{
     class Person
    {
        
        public string Name { get; set; }
        
        public int Age { get; set; }
        
        public  string Gender { get; set; }

        public Person(string name, int age,string gender)
        {
            this.Name = name;
            this.Age = age;
            this.Gender = gender;
        }

        public Person():this("Jane Doe",33,"Female")
        {

        }
        public virtual void Introduce()
        {
            Console.WriteLine($"Hi, I'm {this.Name}, a {this.Age} years old {this.Gender}");
        }

        public virtual void GetGoal()
        {
            Console.WriteLine("My goal is: Live for the moment!");
        }

    }

}

File: DSA-2019/week-02/day-1/person.py
class Person():
    def __init__(self, name="Jane Doe", age="30", gender="female"):
        self.name = name
        self.age = age
        self.gender = gender

    def introduce(self):
        return f" Hi, Im {self.name}, a {self.age} year old {self.gender} "

    def get_goal(self):
        return "My goal is: Live for the moment!"


person = Person("Jane Doe", "30", "female")
